packages/schema-dot-org-types/src/index.js

```diff
@@ -0,0 +1,5 @@
+//@flow
+export {VideoObject} from './generated/video-object';
+export {Place} from './generated/place';
+export {PostalAddress} from './generated/postal-address';
+export {Event} from './generated/event';
```

Comment:
I'll add a comment. It might be a goal to generate the index. I just wanted to export types that are actually needed.